app/views/checks/_boolean_money_with_frequency.slim

```diff
@@ -1,14 +1,12 @@
-- conditional ||= false
 .govuk-summary-list__row
   dt.govuk-summary-list__key.wide
     = label_text
 
   dd.govuk-summary-list__value
-    - if conditional
-      div class="govuk-!-margin-bottom-1"
-        = yes_no_boolean(boolean)
+    div class="govuk-!-margin-bottom-1"
+      = yes_no_boolean(boolean)
 
-    - if !conditional || boolean
+    - if boolean
```

Comment:
Just looking at this - wondering if we should rename 'boolean' as my understanding here is it is more about the relevancy of the answer i.e. if they say 'yes', then the value is relevant. This did confuse me a bit. I might be misunderstanding the flow. Maybe this will all be refactored to be clearer once we have removed all the conditional behaviour.

Reply:
Have implemented something based on your suggestion - the name 'boolean_money_with_frequency' didn't mean much to me either